### PR TITLE
Change layout from admin setting of CMS, Categories, Product pages

### PR DESCRIPTION
**Pull request by speedy008**
## Purpose of this pull request

This pull request (PR) contains details of Changing layout from admin setting of CMS, Categories, Product pages

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.html
- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/layouts/xml-instructions.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-page) files; all other customizations are performed in the [page configuration]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-conf) or [generic layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-gen) files. 
 
+## Change layout using admin configuration of below pages
+
+* CMS pages
+* Categories
+* Products
+
+Above admin settings, under design tab you can change the layout of CMS Page, Categories Pages and Products Pages.
```
Comment (osrecio) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
Above admin settings, under design tab you can change the layout of CMS Page, Category Page and Product Page.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -14,6 +14,16 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 
 
+## Change layout using admin configuration of below pages
+
+* CMS pages
+* Categories
+* Products
+
+Above admin settings, under design tab you can change the layout of CMS Page, Categories Pages and Products Pages.
```
Comment (osrecio) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
Above admin settings, under design tab you can change the layout of CMS Page, Category Page and Product Page.
```

**Comment by lorikrell** [comment]
@erikmarr - This content is currently documented within the Magento Admin User Guide: https://docs.magento.com/m2/ce/user_guide/catalog/settings-advanced-design.html

**Comment by erikmarr** [comment]
@speedy008 thanks for your contribution; however, we do not accept content that duplicates information in the admin user guide. I would recommend we close this PR or you can include a link to the Admin user guide topic: https://docs.magento.com/m2/ce/user_guide/catalog/settings-advanced-design.html somewhere in the XML instructions topic.

**Comment by dobooth** [comment]
Closing per EMarr.

**Pull request closed without merging**